internal/ent/schema/organization.go

```diff
@@ -32,15 +32,38 @@ func (Organization) Fields() []ent.Field {
 			NotEmpty().
 			Annotations(
 				entgql.OrderField("name"),
+				entgql.Skip(entgql.SkipWhereInput),
+			),
+		field.String("description").
+			Comment("An optional description of the Organization").
+			Optional().
+			Annotations(
+				entgql.Skip(entgql.SkipWhereInput),
+			),
+		field.UUID("parent_organization_id", uuid.UUID{}).Default(uuid.New).Unique().Optional().Immutable().
```

Comment:
This shoudn't have a default value, and should be nullable for a `root` org. I don't think it needs to be unique either, a root org could have multiple child orgs. 
```suggestion
		field.UUID("parent_organization_id", uuid.UUID{}).
```